packages/oui/_reboot.less

```diff
@@ -0,0 +1,473 @@
+@import "./_settings";
+@import '../oui-color/_variables';
+@import '../oui-typography/_variables';
+
+// Manually forked from Bootstrap 4 to update variables and port it to LESS.
```

Comment:
Could you have imported Bootstrap 4 Reboot and have all the overwritting properties here instead ? This is not maintenable

Reply:
They're don't use LESS anymore. That's what I explained a bit further :)

Reply:
Or we could go with `normalize.css` with some extends too

Reply:
I like it !